tests/pipeline-all-amps/repo/pom.xml

```diff
@@ -112,32 +123,25 @@
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
                                 </artifactItem>
-                                <!--<artifactItem>
+                                <!-- Do not comment out, as this will cause problems in the Single Pipeline. -->
+                                <artifactItem>
                                     <groupId>org.alfresco.services.sync</groupId>
                                     <artifactId>alfresco-device-sync-repo</artifactId>
                                     <version>${alfresco.desktop-sync.version}</version>
                                     <type>amp</type>
                                     <overWrite>false</overWrite>
                                     <outputDirectory>${project.build.directory}/amps</outputDirectory>
-                                </artifactItem>-->
-                                <!-- TODO uncomment this amp once https://issues.alfresco.com/jira/browse/MNT-21648 is solved -->
-<!--                                <artifactItem>-->
-<!--                                    <groupId>org.alfresco</groupId>-->
-<!--                                    <artifactId>alfresco-document-transformation-engine-repo</artifactId>-->
-<!--                                    <version>${alfresco.transformation-engine.version}</version>-->
-<!--                                    <type>amp</type>-->
-<!--                                    <overWrite>false</overWrite>-->
-<!--                                    <outputDirectory>${project.build.directory}/amps</outputDirectory>-->
-<!--                                </artifactItem>-->
-                                <!-- TODO uncomment this amp once https://alfresco.atlassian.net/browse/APPS-702 is solved -->
-<!--                                <artifactItem>-->
-<!--                                    <groupId>org.alfresco</groupId>-->
-<!--                                    <artifactId>alfresco-governance-services-enterprise-repo</artifactId>-->
-<!--                                    <version>${alfresco.ags.version}</version>-->
-<!--                                    <type>amp</type>-->
-<!--                                    <overWrite>false</overWrite>-->
-<!--                                    <outputDirectory>${project.build.directory}/amps</outputDirectory>-->
-<!--                                </artifactItem>-->
+                                </artifactItem>
+                                <!-- Do not comment out, as this will cause problems in the Single Pipeline. -->
+                                <!-- TODO fix test failure APPS-702 -->
+                                <artifactItem>
+                                    <groupId>org.alfresco</groupId>
+                                    <artifactId>alfresco-governance-services-enterprise-repo</artifactId>
```

Comment:
While cleaning this up, `alfresco-governance-services-enterprise-repo` amp appears two times :)

Reply:
Thanks Ancuta, did not see the comment for some reason. Has been included in the next commit.